src/widgets/modalChildren/NewConfirmModal.js

```diff
@@ -72,10 +72,12 @@ export const NewConfirmModal = props => {
           )}
         </View>
       </View>
-    </>
+    </Modal>
   );
 };
 
+export default NewConfirmModal;
```

Comment:
We stylistically opt to not use default exports, only named (which the component already is when declared).